packages/components/src/components/hds/card/container.hbs

```diff
@@ -2,6 +2,14 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+{{!
+  Dynamically generating an HTML tag in Ember creates a dynamic component class (with the corresponding tagName), while rendering
+  a plain HTML element requires less computing cycles for Ember (you will notice it doesn't add the `ember-view` class to it).
+}}
+{{#if (eq this.componentTag "div")}}
+  <div class={{this.classNames}} ...attributes>{{yield}}</div>
+{{else}}
+  {{#let (element this.componentTag) as |Tag|}}
+    <Tag class={{this.classNames}} ...attributes>{{yield}}</Tag>
```

Comment:
@shleewhite So the way this is done now, even though only 2 choices are currently permitted (`div` or `li`) I'm not hard coding the optional tag choice of `li`. So if I add `role="listitem"` to it if consumers set the tag to `li` I'll have to refactor this or add some additional logic. I can look to do that though or else we could mention the issue in the documentation. What do you think?

Reply:
Ok actually it's not difficult to add. I just need to update the related test as well. Will push up when done.